src/app/components/views/home/home.component.scss

```diff
@@ -0,0 +1,88 @@
+
```

Comment:
- Can't we manage some properties using bootstrap
- Is the home screen responsive

Reply:
ok noted